googler

```diff
@@ -646,6 +658,9 @@ debug = args.debug
 keywords = args.keywords
 insite = args.insite
 noninteractive = args.noninteractive
+if args.json:
+    output_format = 'json'
+    noninteractive = True
```

Comment:
JSON output is most likely piped, I don't think anyone would rather read JSON in interactive use. Also, our prompt uses `raw_input` (python2) or `input` (python3) which writes stdout, totally screwing up piping.

There's always `--count` and `--start` to handle pagination.